src/notFound/NotFound404.jsx

```diff
@@ -3,24 +3,28 @@ import { useHistory } from 'react-router-dom';
 import { Button } from '../components/Buttons/index';
 import { GridNotFound } from '../components/Grid';
 import { Title } from '../components/Text';
-import { NotFoundWrapper, NotFoundButtonCE } from './notfound.style';
+import { NotFoundStyles, ButtonStyles } from './notFound.style';
 
 export const NotFound404 = () => {
   const history = useHistory();
+
+  const handleGoBack = () => history.goBack();
```

Comment:
why not set such a short method in line?

Reply:
Basically for consistency reasons. Now, this is a very opinionated way of working, you already know that term, right?

I also used to put the small functions inline and Chirstos mentioned we should have just one way and after using it I agree it is better, that way one doesn't have to think what is consider long or short. Basically I think one should pick a way and stick to it, that way the code looks the same in every component and one know what to expect. But again, it is just *an* opinion.

Reply:
ok.  I understand that. Sometimes its confusing for me to understand that and also the "rule" of writing less code as possible, you know.   sometimes s does not go together i have the feeling. that's where i need  to learn to 
differentiate

Reply:
If one have some handlers inline and some not, there could cases where you have several buttons in the same component and one have to scroll up and down to check all the onClicks, because some would be inline and some would be on the body of the function.

Reply:
Yes, that is normal. My first advice (and again I have to learn that too), would be not to think in terms of writing less code, but writing *clear* code. I would say that is a much better Ziel to have.

Reply:
Because sometimes, a lot of times, less code just makes the code obscure. Like with a lot of "one-liners" you can find in Stackoverflow. Sure, they can be clever, but if I have to spend 3 minutes to understand it what would I use it?

Reply:
I know to little to say whats better, it makes sense what you said , definitely. but i know not enough from the other side. (you know like performance issues or what ever). You mention a realy good point here. Think we will learn with experience